Common Functions/splice-functions/src/main/java/com/splicemachine/utils/RightPad.java

```diff
@@ -0,0 +1,36 @@
+package com.splicemachine.utils;
```

Comment:
The package name should be a custom package like: com.splicemachine.tutorials.utils.